README.md

```diff
@@ -222,6 +291,40 @@ heapq.heappop(nums) # Pop an element from the heap
 # and the returned value from that function is then used to rank that element in the heap
 heapq.nlargest(k, iterable, key = fun)
 heapq.nsmallest(k, iterable, key = fun)
+
+
+#Max heap in python 
+
+#By default heapq in python is min heap, 
+#if we want to use max heap we can simply invert the value of the keys and use heapq. For example, turn 1000.0 into -1000.0 and 5.0 into -5.0.
+
+#The easiest and ideal solution
+#Multiply the values by -1
+
+#All the highest numbers are now the lowest and vice versa.
+
+#Just remember that when you pop an element to multiply it with -1 in order to get the original value again.
+
+#Example: 
+
+import heapq
+heap = []
+heapq.heappush(heap, 1*(-1))
```

Comment:
heapq.heappush() function already mentioned above with example. this is a duplicate.

Reply:
this heappush() is an example for maxHeap, were we are multiplying the number with -1 before pushing , prev example was for min heap.

Reply:
Okay we can work with that.